pandas/core/groupby.py

```diff
@@ -99,9 +99,11 @@
     func : function
         A callable that takes a {input} as its first argument, and
         returns a dataframe, a series or a scalar. In addition the
-        callable may take positional and keyword arguments
-    args, kwargs : tuple and dict
-        Optional positional and keyword arguments to pass to ``func``
+        callable may take positional and keyword arguments.
+    args : tuple
+        Optional positional and keyword arguments to pass to ``func``.
```

Comment:
This are only the positional ones, and below the keyword ones. 

I think it would also be a good idea to add an example of this in the Examples section